keps/sig-auth/draft-20180620-windows-group-managed-service-accounts.md

```diff
@@ -0,0 +1,218 @@
+---
+kep-number: 0
+title: Windows Group Managed Service Accounts for Container Identity
+authors:
+  - "@patricklang"
+owning-sig: "wg-container-identity"
```

Comment:
The docs note that a SIG needs to own this. See https://github.com/kubernetes/community/blob/master/keps/0001-kubernetes-enhancement-proposal-process.md#kep-metadata

Would sig-auth or sig-windows be a good owner?